pkg/publisher/publisher.go

```diff
@@ -96,7 +86,16 @@ func New(ctx context.Context) (Publisher, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "publisher: unable to obtain EC2 service client")
 	}
-	clusterID := getClusterID(ec2Client)
+	clusterID, err := ec2Client.GetClusterTag("CLUSTER_ID")
+	if err != nil || clusterID == "" {
+		log.Infof("Failed to obtain cluster-id, fetching name.  %v", err)
+		clusterID, err = ec2Client.GetClusterTag("Name")
+		if err != nil || clusterID == "" {
+			log.Infof("Failed to obtain cluster-id or name, defaulting to 'k8s-cluster'.  %v", err)
+			clusterID = "k8s-cluster"
+		}
+	}
+	log.Infof("Using cluster ID %s", clusterID)
```

Comment:
I think the changes from #856 have been accidentally reverted in this file. That's the lines 56-63, 99 and the `getClusterID` function.

Reply:
Not sure what happened here. I reverted it back. Thanks for catching !